tradeexecutor/ethereum/universe.py

```diff
@@ -36,7 +36,7 @@ def create_pair_universe(web3: Web3, exchange: Optional[Exchange], pairs: List[T
         assert p.base.address != p.quote.address
         assert p.fee is not None, f"Pair missing fee {p}"
         dex_pair = DEXPair(
-            pair_id=int(p.get_identifier(), 16),
+            pair_id=p.internal_id or int(p.get_identifier(), 16),
```

Comment:
I don't think this is as proper fix. We should investigate why there the input is wrong in the first place.

Reply:
If I use only `pair_id = p.internal_id` instead of `pair_id=p.internal_id or int(p.get_identifier(), 16)`, there is an error when the internal_id is None -> 'Internal id needed to be hashable`

https://github.com/tradingstrategy-ai/trade-executor/blob/9d9e0a97b128abcc123910e68cc3235d39d00082/tradeexecutor/state/identifier.py#L181-L182